fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -317,7 +320,13 @@ public void cancelLoadJob(CancelLoadStmt stmt, boolean isAccurateMatch) throws D
                 }
             } else {
                 for (Map.Entry<String, List<LoadJob>> entry : labelToLoadJobs.entrySet()) {
-                    if (entry.getKey().contains(stmt.getLabel())) {
+                    String label = stmt.getLabel();
+                    // Compatible with historical issues
+                    if (!label.contains("%")) {
```

Comment:
This should be done in analyze() of CancelLoadStmt.

Reply:
done